posthog/cdp/templates/tiktok/template_tiktok_ads.py

```diff
@@ -0,0 +1,115 @@
+from posthog.cdp.templates.hog_function_template import HogFunctionTemplate
+
+template: HogFunctionTemplate = HogFunctionTemplate(
+    status="alpha",
+    type="destination",
+    id="template-tiktok-ads",
+    name="TikTok Ads Conversions",
+    description="Send conversion events to TikTok Ads",
+    icon_url="/static/services/tiktok.png",
+    category=["Advertisement"],
+    hog="""
+let body := {
+	'event_source': 'web',
+	'event_source_id': inputs.pixelId,
+	'data': [
+		{
+			'event': inputs.eventName,
```

Comment:
I wonder if we should kick off a "tabs" vs "spaces" debate :D

I'm rather impartial, though in the past I think we used 4 spaces for most functions.

Reply:
replaced all tabs with spaces